cli-flags.js

```diff
@@ -0,0 +1,159 @@
+"use strict";
+
+module.exports = {
```

Comment:
Is this needed? What problem does it solve?

Reply:
the file is made to privde the flags for the bundle analyzer plugin, had kept it in the root directory for now. Will move it to appropriate place.

Reply:
Please let me know if somethings wrong, or how to proceed further.